setup.py

```diff
@@ -42,7 +42,8 @@
         "pycrypto==2.6.1",
         "PyYAML==3.12",
         "shyaml==0.5.0",
-        "ansible==2.3.0.0"
+        "ansible==2.3.0.0",
+        "boto3==1.4.4"
```

Comment:
I don't think it needs to be. The setup.py and requirements.txt are separate concerns. setup.py adds the dependancies for the installation of the module. The requirements.txt seems to be more concerned with the build process. It is not leveraged when the module is `pip install`ed. Perhaps @jmound  can fill us in more on the use or this requirements.txt